src/NuGet.Clients/NuGet.SolutionRestoreManager/BrokeredServicesUtility.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Shell.ServiceBroker;
+
+namespace NuGet.SolutionRestoreManager
+{
+    internal class BrokeredServicesUtility
```

Comment:
[CA1052](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1052?view=vs-2019) violation.

Reply:
I made the class static.